frontend/src/app/modules/common/editable-toolbar-title/editable-toolbar-title.sass

```diff
@@ -1,21 +1,57 @@
 .editable-toolbar-title--container
-  display: flex
+  display: grid
   align-items: center
+  grid-template-columns: auto auto 1fr
+
+  &::before
+  &::after
+    display: block
+    content: ''
+
+  &::before
+    grid-area: 1/1
+
+  &::after
+    grid-area: 1/3
 
 .editable-toolbar-title--fixed
   padding: 0
   cursor: default
 
 .editable-toolbar-title--save
+  grid-area: 1/1
+
   span:before
     color: #5F5F5F
 
   &:hover span:before
     color: darken(#5F5F5F, 20%)
 
 .editable-toolbar-title--input
-  padding-left: 5px
   color: #5F5F5F
 
+  &:focus
+    border-color: #999
+    box-shadow: 0 0 0 1px #999
```

Comment:
This is to get a bigger border on when editing. I'm unsure if you want to go through with this, since it would be the only place where it is different.

We could also make this blue, in which case it would be even more different from the rest of the system. That's fine, as long as you know in which design direction you'd like to be going.